scripts/mkcloud

```diff
@@ -230,7 +232,8 @@ function sshrun()
 EOF
     env|grep -e ^debug_ -e ^pre_ -e ^want_ -e ^net_ -e ^nodenumber -e ^clusterconfig | sort >> $mkcconf
 
-    $scp $qa_crowbarsetup $mkcconf root@$adminip:
+    #TODO TH: discuss if the file extension should be part of the variable or not (like it is now)
```

Comment:
So if `cloudversion=GM5` and `scenario=whatever` it would take file `./scenarios/cloud5/whatever.yaml`, right?
Do you think its also necessary to be able to distinguish between the different versions of the same release? (devel, GM, GM+up)